src/plugins/discover/public/application/apps/main/components/doc_table/lib/row_formatter.scss

```diff
@@ -0,0 +1,7 @@
+// Special handling for images coming from the image field formatter
+// See discover_grid.scss for more explanation on those values
+.rowFormatter__value img {
+  vertical-align: middle;
+  max-height: lineHeightFromBaseline(2) !important;
+  max-width: 500px !important;
+}
```

Comment:
just loud thinking, how about assigning a class in the field formatter code, 

https://github.com/elastic/kibana/blob/d6badd8715ecf37fb7899f85a671be48031e67bd/src/plugins/field_formats/common/converters/url.ts#L130

using this class to have single source of truth for the formatting in classic and new table?

Reply:
I have mixed feelings around that. While I agree a CSS class is nicer to match than a tag, it means the Discover plugin (specifically it's CSS) will suddenly need to have knowledge about the internals of a completely other plugin and how that field formatter internally renders. So I am on the fence if I would prefer that more or less. Your thoughts after having my context?